app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    @current_user ||= User.first
+  end
```

Comment:
- [ ] I don't know but you don't have the user form view to create a user for the authentication and this results in
 this kind of error cause according to the project, all the models depend o the user model since they all have user_id or author_id foreign key. Please see how to make your application view accessible from the browser. 👍🏻 
 
 
 
![image](https://github.com/Microverse-Fullstack-Program/rails-blog-app/assets/79058364/9df1faaf-8997-4af0-a149-12ea52cc1a8e)